deepspeed/module_inject/auto_tp_model_utils.py

```diff
@@ -60,6 +60,14 @@ def build_bloom_alibi_tensor(attention_mask: torch.Tensor, num_heads: int, dtype
     else:
         return alibi.reshape(batch_size * num_heads, 1, seq_length).to(dtype)
 
```

Comment:
Formatting wise you should just need additional new lines above and below this function.

Reply:
fixed.